src/sentry/static/sentry/app/utils/errorHandler.tsx

```diff
@@ -1,29 +1,33 @@
 import React from 'react';
+import * as Sentry from '@sentry/browser';
+
 import RouteError from 'app/views/routeError';
 
 export default function errorHandler(Component) {
   class ErrorHandler extends React.Component {
-    static getDerivedStateFromError(error) {
-      // Update state so the next render will show the fallback UI.
+    // Update state so the next render will show the fallback UI.
+    static getDerivedStateFromError(error: Error) {
       return {
         hasError: true,
         error,
       };
     }
 
+    // we are explicit if an error has been thrown since errors thrown are not guaranteed
+    // to be truthy (e.g. throw null).
     state = {
-      // we are explicit if an error has been thrown since errors thrown are not guaranteed
-      // to be truthy (e.g. throw null).
       hasError: false,
       error: null,
     };
 
-    componentDidCatch(error, info) {
+    componentDidCatch(error: Error, info: {componentStack: string}) {
       // eslint-disable-next-line no-console
       console.error(
         'Component stack trace caught in <ErrorHandler />:',
         info.componentStack
       );
+
+      Sentry.captureException(error);
```

Comment:
@leedongwei This should already be emitted here https://github.com/getsentry/sentry/blob/37dc9190737f4a754b3c58447497a57b8a82ca55/src/sentry/static/sentry/app/views/routeError.jsx#L50

Reply:
Unless this does something else 🤔

Reply:
Yeah I think @dashed is right